### PR TITLE
Lps 76955 pr

### PR DESCRIPTION
**Pull request by gregory-bretall**
PR Test Results: https://github.com/gregory-bretall/liferay-portal/pull/29#issuecomment-367860994

Resent from https://github.com/gregory-bretall/liferay-portal/pull/29

@SpencerWoo

### CODE REVIEW

**Comment by lipusz** [comment]
Hi Jonathan,

Please make sure it goes through the Application Security team too.

Thanks!

**Comment by lipusz** [comment]
Carlos Sierra (csierra) handles usually the WS related things in the team.

**Comment by jonathanmccann** [comment]
@lipusz Thanks for letting me know! I'll be sure to pass it through him.

File: portal-web/docroot/html/portal/api/jsonws/action.jsp [hunk 1]
```diff
@@ -349,6 +349,17 @@ String signature = ParamUtil.getString(request, "signature");
 						</aui:field-wrapper>
 
 					<%
+					}
+					else if (methodParameterTypeClassName.startsWith("com.")) {
```
Comment (jonathanmccann) [inline portal-web/docroot/html/portal/api/jsonws/action.jsp]: @SpencerWoo Wouldn't this break if we wanted to add a parameter whose path starts with "org" or something similar? I may be misunderstanding what this is doing though.

Comment (SpencerWoo) [reply portal-web/docroot/html/portal/api/jsonws/action.jsp]: @jonathanmccann 
Here we're creating different GUI input fields based on the _expected input types_.

For the line here we're looking to trigger when the input type is an object so we're using "com.". 
 The parameter input content can be anything but we're trying to match object parameter types.

However if there are JSONWS methods that takes object parameters but don't start with "com." as a type then it's true that we wouldn't interpret them as an object and the else condition would execute giving the current behavior via GUI.  If this is the case I'd like to talk about it but from looking around at the JSONWS methods I thought "com." was the best to match object parameter types.

**Comment by jonathanmccann** [comment]
https://github.com/csierra/liferay-portal/pull/312

**Pull request closed without merging**